### PR TITLE
minutes from Feb 20 meeting

### PR DESCRIPTION
**Pull request by mbonsma**

### CODE REVIEW

File: meetings/Minutes_2018-02-20.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## Minutes - Jan 20, 2018
+
+1. **RQM Course**
+ * New material: simulations / bootstrapping & permutations (Lindsay & Luke), mixed effects modelling, time series analysis / simulating time series, practical quantitative stats (Lindsay - hypothesis testing, fit statistics)
```
Comment (lwjohnst86) [inline meetings/Minutes_2018-02-20.md]: Ah, nice I like that bootstrapping and permutations, good good.

File: meetings/Minutes_2018-02-20.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * Have another meeting soon and invite James from EEB
+ * Ask if we can have two course instructors (Lindsay and Ahmed or James) and two TAs (Madeleine and Lina or Ahmed)? 
+ * Otherwise, 4 TAs, equal division of hours: Lindsay, Madeleine, Ahmed, and James
+ * ACTION: Ahmed talk to James
+ * ACTION: Joel email Martin and Asher 
+ * Publishing a paper: publish on bioarxiv, or in this other journal that Data Carpentry publishes in - "International Journal of Digital Curation" (no publication cost)
+
+2. **SWC**
+ * Past workshop was generally well-received.
+ * A new workshop: one just for women.
```
Comment (lwjohnst86) [inline meetings/Minutes_2018-02-20.md]: very good idea, I like that! You guys talk about collaborating with Ladies Who Code? (if that's the right name of the group)

Comment (mbonsma) [reply meetings/Minutes_2018-02-20.md]: Ladies Learning Code? I actually taught for them last week, they'd definitely be good to talk to about this. Although their focus is less on programming for research.

File: meetings/Minutes_2018-02-20.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+4. **Misc**
+ * Should we moderate posts from strangers better? 
+ * In contributing:
+     * Mention the types of things that are okay
+ * Make a template for issues (Lina) 
+     * Name, contact info, issue or event or volunteer or job opportunity, details, why you think UofT Coders can help or would be interested 
+ * In March, announce positions and election
+ * Finalize new positions, maybe make a position that will be unfilled by us?
+     * Lesson maintainer / code reviewer?
```
[the remark this answers is not in the document]
Comment (mbonsma) [reply meetings/Minutes_2018-02-20.md]: Yeah, I think in addition to technical admin, assuming that @QuLogic will be continuing to keep an eye on things.

**Pull request merged**